libbeat/docs/http-endpoint.asciidoc

```diff
@@ -0,0 +1,149 @@
+//////////////////////////////////////////////////////////////////////////
+//// This content is shared by all Elastic Beats. Make sure you keep the
+//// descriptions here generic enough to work for all Beats that include
+//// this file. When using cross references, make sure that the cross
+//// references resolve correctly for any files that include this one.
+//// Use the appropriate variables defined in the index.asciidoc file to
+//// resolve Beat names: beatname_uc and beatname_lc.
+//// Use the following include to pull this content into a doc file:
+//// include::../../libbeat/docs/http-endpoint.asciidoc[]
+//////////////////////////////////////////////////////////////////////////
+
+[[http-endpoint]]
+== HTTP Endpoint
+
+experimental[]
+
+{beatname_uc} can expose internal metrics through a HTTP endpoint. For security
+reasons the endpoint is disabled by default.
+
+The HTTP endpoint has the following configuration settings:
+
+`http.enabled`:: (Optional) Enable the http endpoint. Default is `false`.
+`http.host`:: (Optional) Bind to this hostname or IP address.
+It is recommended to use only localhost. Default is `localhost`
+`http.port`:: (Optional) Port on which the HTTP endpoint will bind. Default is `5066`.
+
+[float]
+=== Paths
```

Comment:
I'm not sure this is the correct title for this section. We use "paths" in other places to describe config options for setting log file locations. I think we should avoid using "paths" here because it looks too much like a config option.

Unless I'm misunderstanding something here, these endpoints seem similar to what we provide in Logstash. If so, it might be better focus on what the endpoints return (similar to what we do in the Logstash Reference under this section: https://www.elastic.co/guide/en/logstash/current/node-stats-api.html). Since we only have a couple of endpoints, though, you probably don't need a whole structure like we have in the LS Reference (might seem over-engineered at this point).